collectors/cgroups.plugin/cgroup-network.c

```diff
@@ -447,6 +454,23 @@ void detect_veth_interfaces(pid_t pid) {
         goto cleanup;
     }
 
+    // host ifaces == guest ifaces => we are still in the host namespace
+    // and we can't really identify which ifaces belong to the cgroup (e.g. Proxmox VM).
+    if (host->count == cgroup->count) {
+        int m = 0;
+        for (h = host; h; h = h->next) {
+            for (c = cgroup; c; c = c->next) {
+                if (h->ifindex == c->ifindex && h->iflink == c->iflink) {
+                    m += 1;
+                    break;
+                }
+            }
+        }
+        if (host->count == m) {
```

Comment:
Please, convert the variable `m` to `unsigned int` to remove this warning:

```sh
collectors/cgroups.plugin/cgroup-network.c: In function 'detect_veth_interfaces':
collectors/cgroups.plugin/cgroup-network.c:469:25: warning: comparison of integer expressions of different signedness: 'unsigned int' and 'int' [-Wsign-compare]
  469 |         if (host->count == m) {
```

Reply:
@ilyam8 I tested on Linux distributions with both cgroup versions, I am only waiting for a commit to fix this warning.

Reply:
fixed thanks